docs/dev/Lua API.rst

```diff
@@ -1834,6 +1834,18 @@ Items module
 
   Marks the given item for trade at the given depot.
 
+* ``dfhack.items.canMelt(item)``
+
+  Returns true if the item can be designated for melting.
+
+* ``dfhack.items.markForMelting(item)``
+
+  Marks the given item for melting.
```

Comment:
perhaps add something to the effect of: "if it's not already marked for melting"